File: students/students/src/MON.Models/Institution/InstitutionInfoModel.cs
namespace MON.Models.Institution
{
    public class InstitutionInfoModel : InstitutionCacheModel
    {
        public string Region { get; set; }
        public string Municipality { get; set; }
        public int? MunicipalityId { get; set; }
        public string Town { get; set; }
        public int? TownId { get; set; }
    }
}

File: students/students/src/MON.SPA/src/models/institution/institutionInfoModel.js
export class InstitutionInfoModel{
    constructor(obj = {}){
        this.id = obj.id || null;
        this.name = obj.name || '';
        this.region = obj.region || '';
        this.regionId = obj.regionId || null;
        this.municipality = obj.municipality || '';
        this.municipalityId = obj.municipalityId || null;
        this.town = obj.town || '';
        this.townId = obj.townId || null;
    }
}